public/registration_form.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <title>Hospital Registration</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="../assets/css/registration_form.css">
+    <script type="text/javascript" src="../assets/js/registration_form.js"></script>
```

Comment:
Always add script at the end of the body

Reply:
Done the changes @991rajat .